docs/RESOURCES.md

```diff
@@ -1,20 +1,12 @@
 # Resources
 
-To get help if you're having trouble, you can use one of the following resources:
+- Learning resources
+    - [Get Started](https://ballerina.io/learn/get-started/)
+    - [Ballerina By Example](https://ballerina.io/learn/by-example/)
+    - [Learn Ballerina](https://ballerina.io/learn/)
```

Comment:
Add this as last one as it is the page which contains all the others as well.

Reply:
Updated.